pkg/services/alerting/eval_context.go

```diff
@@ -31,9 +31,10 @@ type EvalContext struct {
 	PrevAlertState  m.AlertStateType
 
 	Ctx context.Context
+	Cfg *setting.Cfg
```

Comment:
I added global cfg to eval context so there is bit less propagation needed as this context is used in most places in alerting.